packages/smooth_app/lib/background/background_task_image.dart

```diff
@@ -1,9 +1,12 @@
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
```

Comment:
Never seen that path before but when CI doesn't complain I'm fine with that

Reply:
@M123-dev Then you never _started_ a localized page, because it's all over the place (96 occurrences...).
For some reason my Android Studio seems to have problems suggesting that `import`; that's how I learned it the hard way and copied/pasted again and again for each new localized page.

Reply:
I guess then my first assumption was wrong then, I have the same problem with the import. I just thought it was a different path